### PR TITLE
Fix some problems

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

File: lib/Lexer.php [hunk 1]
```diff
@@ -74,21 +74,19 @@ public function lex(string $docblock): Tokens
         );
 
         $tokens = [];
-        $prevChunk = null;
         foreach ($chunks as $chunk) {
             [ $value, $offset ] = $chunk;
             $tokens[] = new Token(
                 $offset,
-                $this->resolveType($value, $prevChunk),
+                $this->resolveType($value),
```
Comment (szepeviktor) [inline lib/Lexer.php:81]: `$prevChunk` was unused. Is it all right?

Comment (dantleech) [reply lib/Lexer.php:81]: yep

**Comment by dantleech** [comment]
Thanks for the effort - to be honest there are multiple cs/stan issues at the moment as this is library is a work-in-progress - I will get the build passing soon :)

**Comment by dantleech** [comment]
Thanks

**Pull request merged**